retriever/v2/server_test.go

```diff
@@ -81,7 +81,7 @@ func TestRetrieveBlob(t *testing.T) {
 	server := newTestServer(t)
 	data := codec.ConvertByPaddingEmptyByte(gettysburgAddressBytes)
 	retrievalClient.On(
-		"GetBlob",
+		"GetBlobWithProbe",
```

Comment:
Presumably it should be `GetBlob`, given your recent changes?

Reply:
fixed